src/main/java/earl/gui/DialogBox.java

```diff
@@ -0,0 +1,65 @@
+package earl.gui;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
```

Comment:
I like how you imported all of the classes explicitly instead of using wildcards even though you have multiple import statements. The ordering of the import statements are also consistent!